lib/functions/url.js

```diff
@@ -114,19 +114,19 @@ module.exports = function(options) {
     }
 
     // Read data
-    var str = fs.readFileSync(found, 'utf8');
```

Comment:
that's a global

Reply:
`buf` has been declared [here](https://github.com/mockee/stylus/blob/96c02deccf6d64160f415b7ca3302811d9b846e9/lib/functions/url.js#L94).